File: server/router/normal/gost_node.go
package normal

import (
	"github.com/gin-gonic/gin"
	"server/controller/normal/gost_node"
	"server/global"
	"server/router/middleware"
)

func InitGostNode(group *gin.RouterGroup) {
	g := group.Group("gost/node", middleware.Auth(global.Jwt))
	g.POST("list", gost_node.List)
	g.POST("cleanPort", gost_node.CleanPort)
	g.POST("create", gost_node.Create)
	g.POST("delete", gost_node.Delete)
	g.POST("update", gost_node.Update)
	g.POST("page", gost_node.Page)
}

File: web/src/api/normal/gost_node.js
import request from "../../request/index.js";

const baseUrl = '/v1/normal/gost/node'

export const apiNormalGostNodeList = (data) => {
    return request.request({
        url: `${baseUrl}/list`,
        method: 'POST',
        data
    })
}

export const apiNormalGostNodeCleanPort = (data) => {
    return request.request({
        url: `${baseUrl}/cleanPort`,
        method: 'POST',
        data
    })
}


export const apiNormalGostNodePage = (data) => {
    return request.request({
        url: `${baseUrl}/page`,
        method: 'POST',
        data
    })
}

export const apiNormalGostNodeCreate = (data) => {
    return request.request({
        url: `${baseUrl}/create`,
        method: 'POST',
        data
    })
}

export const apiNormalGostNodeUpdate = (data) => {
    return request.request({
        url: `${baseUrl}/update`,
        method: 'POST',
        data
    })
}


export const apiNormalGostNodeDelete = (data) => {
    return request.request({
        url: `${baseUrl}/delete`,
        method: 'POST',
        data
    })
}
